GiniVisionExample/GiniVisionExample/AppCoordinator.swift

```diff
@@ -48,7 +54,9 @@ final class AppCoordinator: Coordinator {
 extension AppCoordinator: MainViewControllerDelegate {
     
     func main(viewController: MainViewController, didTapStartAnalysis: ()) {
-        
+        let navigationController = UINavigationController(rootViewController: pdfNoResultsViewController)
+        navigationController.navigationBar.barTintColor = .blue
```

Comment:
Does it make sense to set the tint "globally"? Maybe even with `UIAppearance`.

Reply:
Actually it uses a color that is already in `develop`, so I was planning to change once it merges. I think that UIAppearance would be an overkill.